### PR TITLE
new studentName

### PR DESCRIPTION
**Pull request by DmytroTikhostup**
added yourself in list #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,21 @@ Learning course for newbies. I'll add some stuff. I'm bad developer and do stuff
...
+-   Vladislav Pokhyl : VladFlaber
+-   Roman Sokurenko: RomanSokurenko
+-   Evgeniy Dotsenko: EvgenyDotsenko
+-   David Korystov: DavidKorystov
+-   Pavel Borisenko: PavelBorisenkoRD
+-   Vasiliy Gladush: vasyagladush
+
+### Group #4
+
+-   Dmytro Tikhostup: DmytroTikhostup;
```
Comment (VitaliiDenysyuk) [inline README.md:26]: Something wrong with your modification. You have changed all rows in file. It's bad experience. You have to add only one row.

File: README.md [hunk 2]
```diff
@@ -6,16 +6,21 @@ Learning course for newbies. I'll add some stuff. I'm bad developer and do stuff
...
-- Maksim Fomin: fominmaksim
-- Andrew Kaenko: AndrewKaenko
-- Alex Mishenko: AlexMishenko
-- Eduard Valevich: EduardValevich
-- Ivan Lysenko: IvanLsenko
+-   Maksim Fomin: fominmaksim
+-   Andrew Kaenko: AndrewKaenko
+-   Alex Mishenko: AlexMishenko
+-   Eduard Valevich: EduardValevich
+-   Ivan Lysenko: IvanLsenko
```
Comment (alexandrtovmach) [inline README.md:13]: seems like unexpected formatting changes

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]
revert formatting on unrelated changes

File: README.md (hunk 1 above)
Comment (DmytroTikhostup) [reply README.md:26]: This is strange. just added one line. I will correct now

**Pull request closed without merging**